automation/build-bin.ts

```diff
@@ -354,6 +370,39 @@ export async function buildOclifInstaller() {
 	let packOpts = ['-r', ROOT];
 	if (process.platform === 'darwin') {
 		packOS = 'macos';
+		await new Promise((resolve, reject) => {
+			klaw('node_modules/')
+				.on('data', (item: { path: string; stats: Stats }) => {
+					if (!item.stats.isFile()) {
+						return;
+					}
+					if (
+						path.basename(item.path).endsWith('.zip') &&
+						path.dirname(item.path).includes('test')
+					) {
+						console.log('Removing zip', item.path);
+						fs.unlinkSync(item.path);
+					}
+				})
+				.on('end', resolve)
+				.on('error', reject);
+		});
+		// Sign all .node files first
+		await new Promise((resolve, reject) => {
+			klaw('node_modules/')
+				.on('data', async (item: { path: string; stats: Stats }) => {
+					if (!item.stats.isFile()) {
+						return;
+					}
+					if (path.basename(item.path).endsWith('.node')) {
+						await whichSpawn(
+							`productsign --sign "Developer ID Installer: Rulemotion Ltd (66H43P8FRG)" ${item.path}`,
```

Comment:
This should be something like:
```js
await whichSpawn(
    'productsign',
    [
        '--sign',
        'Developer ID Installer: Rulemotion Ltd (66H43P8FRG)',
        item.path,
    ],
);
```

Reply:
This is what makes it shell agnostic (independent of the OS shell being cmd.exe, bash, PowerShell, zsh, etc.) If it took a single long string as argument, it would need to use a shell to parse it. By taking a Javascript array of arguments, it does not need to get any shell involved.